src/modules/pipelines/README.md

```diff
@@ -0,0 +1,184 @@
+| ProductId                  | String |
+| ProductName                | String | ProductName \|\| Product                                                                                    |
+| MeterId                    | String | MeterId \|\| ResourceGuid                                                                                   |
+| MeterName                  | String |
+| PartNumber                 | String |
+| MeterRegion                | String |
+| AdditionalInfo             | Object | AdditionalInfo startswith "{"<br>? parsejson("\{{AdditionalInfo}\}")<br>: parsejson(AdditionalInfo)         |
+| ServiceInfo1               | String |
+| ServiceInfo2               | String |
+| IsAzureCreditEligible      | Bool   | tolower(tostring(IsAzureCreditEligible \|\| IsCreditEligible)) == "true"                                    | Handle both string and boolean values.                  |
+| ChargeType                 | String | ChargeType \|\| "Usage"                                                                                     | Default to "Usage" for classic accounts.                |
+| PricingModel               | String | PricingModel \|\| "OnDemand"                                                                                | Default to "OnDemand" for classic accounts.             |
+| BenefitId                  | String | BenefitId == null<br>? ReservationId : BenefitId                                                            |
+| BenefitName                | String | BenefitName == null<br>? ReservationName : BenefitName                                                      |
+| Term                       | String |
+| Frequency                  | String |
+| CostAllocationRuleName     | String |
+| InvoiceId                  | String |
+| PreviousInvoiceId          | String |
+| BillingPeriodStartDate     | Date   |
+| BillingPeriodEndDate       | Date   |
+| ServicePeriodStartDate     | Date   |
+| ServicePeriodEndDate       | Date   |
+| Quantity                   | Number | Quantity \|\| UsageQuantity                                                                                 |
+| UnitPrice                  | Number | UnitPrice \|\| ResourceRate                                                                                 |
+| EffectivePrice             | Number | EffectivePrice \|\| UnitPrice                                                                               |
+| PayGPrice                  | Number | PayGPrice \|\| UnitPrice                                                                                    |
+| UnitOfMeasure              | String |
+| BillingCurrency            | String | BillingCurrency \|\| BillingCurrencyCode \|\| Currency                                                      |
```

Comment:
Can we normalize on {type}CurrencyCode instead? Also add to notes: ISO 4217 Currency Codes